Workbooks/Workloads/SQL/SQL Server Insights/SQL Server Insights.workbook

```diff
@@ -390,7 +390,7 @@
     {
       "type": 1,
       "content": {
-        "json": "[SQL Insights (preview)](https://learn.microsoft.com/azure/azure-sql/database/sql-insights-overview) will be [retired](https://aka.ms/sql-insights-retirement-azure-update) on **31 December 2024**. Transition to database watcher or another monitoring solution by that date.\r\n\r\n[Database watcher (preview)](https://aka.ms/dbwatcher) is the recommended monitoring solution for scenarios that require low data collection latency, estate-level monitoring, comprehensive monitoring data including query-level details, and low per-database cost for larger environments. At this time, database watcher supports Azure SQL Database and Azure SQL Managed Instance.",
+        "json": "[SQL Insights (preview)](https://learn.microsoft.com/azure/azure-sql/database/sql-insights-overview) has been [retired](https://aka.ms/sql-insights-retirement-azure-update) on **31 December 2024**. Transition to database watcher or another monitoring solution by that date.\r\n\r\n[Database watcher (preview)](https://aka.ms/dbwatcher) is the recommended monitoring solution for scenarios that require low data collection latency, estate-level monitoring, comprehensive monitoring data including query-level details, and low per-database cost for larger environments. At this time, database watcher supports Azure SQL Database and Azure SQL Managed Instance.",
```

Comment:
Please also remove `Transition to database watcher or another monitoring solution by that date.`